### PR TITLE
release: 2.66 changelogs 

### PR DESCRIPTION
**Pull request by ernestl**
Merged master back into changelogs-2.66 branch which is based off tip of release/2.66.
This is required to introduce the commit history that includes the release tagged commit so that it is searchable on the master branch.

**Merge changelogs back into master, please use a regular “merge” to merge it to retain commit history.**

### CODE REVIEW

**Review by zyga — APPROVED** [review]
LGTM with one remark about packaging.

**Review by bboozzoo — APPROVED** [review]
Thanks

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+* Snap components: enable creating Ubuntu Core images that contain components
+* Snap components: handle refreshing components independently of snaps
+* Snap components: handle removing components when refreshing a snap that no longer defines them
+* Snap components: extend snapd Ubuntu Core installation API to allow for picking optional snaps and components to install
+* Snap components: extend kernel.yaml with "dynamic-modules", allowing kernel to define a location for kmods from component hooks
+* Snap components: renamed component type "test" to "standard"
+* Desktop IDs: support installing desktop files with custom names based on desktop-file-ids desktop interface plug attr
+* Auto-install snapd on classic systems as prerequisite for any non-essential snap install
+* Support loading AppArmor profiles on WSL2 with non-default kernel and securityfs mounted
+* Debian/Fedora packaging updates
```
[the remark this answers is not in the document]
Comment (ernestl) [reply NEWS.md:21]: Thanks, will modify this for github tarball release.

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks

**Pull request closed without merging**